less/core/accessibility.less

```
@@ -3,32 +3,33 @@
 .component { position: relative; }
 .block { position: relative; }
 
-// jquery.a11y start
-// Invisible aria-labels
-// --------------------------------------------------
-.aria-label {
-  position: absolute !important;
-  &.relative {
-    position: relative !important;
-  }
-  left: 0 !important;
-  width: auto !important;
-  height: auto !important;
-  overflow: auto !important;
-  color: rgba(0,0,0,0) !important;
-  background: rgba(0,0,0,0) !important;
-  font-size: 1px !important;
-  padding: 0 !important;
-  margin: 0 !important;
-  line-height: normal !important;
-  z-index: 1;
+// Use to hide content visually while keeping it accessible to assistive technologies
+.visually-hidden() {  
```

We should remove the parenthesis to keep the `.visually-hidden` alias available
```suggestion
.visually-hidden {  
```

---

Is there a specific use case to using `.visually-hidden` vs `.aria-label`? Same for `.aria-label-focusable` vs `.visually-hidden-focusable`.